dependencies/maven/deployment_rules_builder.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import sys
+import re
+
+
+def parse_deployment_properties(fn):
+    deployment_properties = {}
```

Comment:
Why is this file `deployment_rules_builder.py` not placed in `graknlabs/deployment`, @vmax ? Isn't this something generic that we want all users of `graknlabs/deployment` to be able to use?

Reply:
@haikalpribadi Makes sense 🤔 The `genrule` part invoking it will still stay in `graknlabs/grakn`, right?

Reply:
yes, the `genrule()` stays in the user's repository (in this case `graknlabs/grakn`